functions/dailyCC/dailyCC.js

```diff
@@ -0,0 +1,124 @@
+const axios = require("axios");
+const {slugs, baseUrl} = require('./constants')
+const {randomIndex, removeFromList} = require('../utilities')
+
+const getQuestion = async (slug)=>{
+	let url = baseUrl + '/' + slug
+	try{
+		let resp = await axios.get(url)
+		return resp.data
+	}
+	catch(err){
+		throw err
+	}
+}
+
+
+const getUsedSlugs = async (firestore)=>{
```

Comment:
Remove firestore since it'll be in its own config. Doesn't need to be an object that's passed here either.